win32ss/user/ntuser/class.c

```diff
@@ -43,19 +43,19 @@ REGISTER_SYSCLASS DefaultServerClasses[] =
     FNID_DESKTOP,
     ICLS_DESKTOP
   },
-  { ((PWSTR)((ULONG_PTR)(WORD)(0x8003))),
+  { ((PWSTR)WC_SWITCH),
     CS_VREDRAW|CS_HREDRAW|CS_SAVEBITS,
     NULL, // Use User32 procs
-    sizeof(LONG),
+    sizeof(LONG_PTR), // See user32_apitest GetSetWindowInt, 0: Pointer to ALTTABINFO
     (HICON)OCR_NORMAL,
     NULL,
     FNID_SWITCH,
     ICLS_SWITCH
   },
-  { ((PWSTR)((ULONG_PTR)(WORD)(0x8000))),
+  { ((PWSTR)WC_MENU),
     CS_DBLCLKS|CS_SAVEBITS|CS_DROPSHADOW,
     NULL, // Use User32 procs
-    sizeof(LONG),
+    16, // See user32_apitest GetSetWindowInt, PopupMenuWndProcW
```

Comment:
What is this referring to?

Reply:
It should in fact be `GetClassInfo` as I decided to move the tests. See https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/apitests/user32/GetClassInfo.c;h=37862c5d1db1540956118911630db9069b570540;hb=636f2e3241795d85e11c4b2fd073f8391d75d20c#l92
And yes, it's `sizeof(ULONG_PTR)` and that is what you get for reviewing a **DRAFT**.
Draft means: not ready for review. I.e. this is WIP and being tested and improved, until it's ready.


-